cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/DetailsInfo/DetailsInfo.scss

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../../../../../styles/variables.scss";
+$details-row-border-color: $grey-05;
+
+.profile-detail-view {
+  .detail-view-content {
+    .view-details-label {
+      cursor: pointer;
+      display: flex;
+      align-items: center;
+
+      .icon-svg {
+        font-size: 1.3rem;
+      }
+
+      > span {
+        color: $blue-03;
+      }
+    }
+
+    .details-content {
+      margin-top: 25px;
+
+      > h5 {
+        margin-bottom: 20px;
+      }
+
+      .details-table {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
```

Comment:
Based on the mocks we don't seem to be having a fixed column length right? Was it decided we have 3 columns all the time?

Reply:
Also the contents are justified as `space-between` but the mocks doesn't seem to be having that.

Reply:
![screen shot 2018-04-25 at 12 46 53 pm](https://user-images.githubusercontent.com/1452845/39269115-c30c339e-4886-11e8-9ea2-3136081f50f5.png)

Reply:
Yes I talked with Lea about making these changes.